cmd/build.go

```diff
@@ -16,7 +16,13 @@ func init() {
 	buildCmd.Flags().BoolP("confirm", "c", false, "Prompt to confirm all configuration options - $FAAS_CONFIRM")
 	buildCmd.Flags().StringP("image", "i", "", "Optional full image name, in form [registry]/[namespace]/[name]:[tag] for example quay.io/myrepo/project.name:latest (overrides --repository) - $FAAS_IMAGE")
 	buildCmd.Flags().StringP("path", "p", cwd(), "Path to the Function project directory - $FAAS_PATH")
+	buildCmd.Flags().StringP("builder", "b", "default", "Buildpacks builder")
```

Comment:
All of the commands have their flags in alphabetical order. I'm not tied to this, but I think it's probably good to be consistent.